pkg/routeagent_driver/handlers/ovn/gateway_dataplane.go

```diff
@@ -170,11 +192,16 @@ func (ovn *Handler) cleanupForwardingIptables() error {
 		"error clearing chain %q", forwardingSubmarinerFWDChain)
 }
 
-func (ovn *Handler) getSubmDefaultRoute() *netlink.Route {
+func (ovn *Handler) getSubmDefaultRoute() (*netlink.Route, error) {
```

Comment:
Please rename this API to a more suitable one. If i understand correctly, this is the nextHop on the `ovn-k8s-mp0` interface and not the default route of the host/submariner itself.

Reply:
Done.